### DISCUSSION TITLE
Rebasing asset paths

### DISCUSSION THREAD

**Discussion by BenjaminWaye** [parcel-bundler/parcel#6349] [Q&A] [answered]
What is the best way to go about rebasing asset paths? Fetching CSS from a remote repo and need to rebase asset paths to dist. Has anyone had any success with this?

**Comment by mischnic** [comment, marked as answer]
A custom resolver plugin might work for this. You can return code (file contents) instead of a filepath

**Reply by BenjaminWaye** [reply]
Are there any guides for writing custom resolvers for parcel?

**Reply by BenjaminWaye** [reply]
By using @mischnic repo https://github.com/mischnic/parcel-resolver-root it is possibly to quickly get a testable custom resolver up and running by simply replacing the code. Thanks @mischnic !
